fv3core/pace/fv3core/stencils/dyn_core.py

```diff
@@ -998,7 +1011,8 @@ def __call__(
         if self._do_del2cubed:
             self._halo_updaters.heat_source.update()
             # TODO: move dependence on da_min into init of hyperdiffusion class
-            cd = constants.CNST_0P20 * self._da_min
+            da_min: float = self._get_da_min()
```

Comment:
AFAIK the float type hint doesn't do anything here. 

```suggestion
            da_min = self._get_da_min()
```

Reply:
It actually is needed so DaCe can parse the callback return value properly